Source/WebCore/dom/TreeScope.cpp

```diff
@@ -572,4 +577,165 @@ ExceptionOr<void> TreeScope::setAdoptedStyleSheets(Vector<RefPtr<CSSStyleSheet>>
     return ensureAdoptedStyleSheets().setSheets(WTFMove(sheets));
 }
 
+struct SVGResouresMap {
+    WTF_MAKE_NONCOPYABLE(SVGResouresMap); WTF_MAKE_FAST_ALLOCATED;
+public:
+    SVGResouresMap() = default;
```

Comment:
I think we can omit this.

Reply:
If I delete this, we get a compilation error like this:
```
unique_ptr.h:728:32: error: no matching constructor for initialization of 'WebCore::SVGResourcesMap'
    return unique_ptr<_Tp>(new _Tp(_VSTD::forward<_Args>(__args)...));
...
WebKitBuild/Debug/usr/local/include/wtf/StdLibExtras.h:596:17: note: in instantiation of function template specialization 'std::make_unique<WebCore::SVGResourcesMap>' requested here
    return std::make_unique<T>(std::forward<Args>(args)...);
                ^
In file included from WebKitBuild/Debug/DerivedSources/WebCore/unified-sources/UnifiedSource160.cpp:5:
dom/TreeScope.cpp:581:26: note: candidate constructor not viable: requires 1 argument, but 0 were provided
    WTF_MAKE_NONCOPYABLE(SVGResourcesMap); WTF_MAKE_STRUCT_FAST_ALLOCATED;
                         ^
```